src/components/Button/StrokeButton.mdx

```diff
@@ -0,0 +1,82 @@
+---
+name: StrokeButton
+menu: Reusable Components
+---
+
+import { Playground, Props } from 'docz'
+import { Toggle as ToggleState } from 'react-powerplug'
+
+import { COLORS } from '../../constants'
+import StrokeButton from './StrokeButton'
+
+# StrokeButton
+
+<Props of={StrokeButton} />
+
+
+## Basic usage
+
+<Playground>
+  <StrokeButton>Default</StrokeButton>
+</Playground>
+
+
+## Sizes
+
+<Playground>
+  <StrokeButton size="xsmall">
```

Comment:
Huh, this one is looking funky 🤔 Commenting as a reminder to investigate later.

<img width="227" alt="Screen Shot 2019-04-07 at 4 14 24 PM" src="https://user-images.githubusercontent.com/17421347/55691192-592eb600-5950-11e9-90f1-52a3967552c7.png">